config.yml

```diff
@@ -3,90 +3,90 @@ commanders:
     date: 2016-05-17
     race: Zerg
     sc2cooplink: https://starcraft2coop.com/commanders/abathur
-    wikilink: https://starcraft.fandom.com/wiki/Abathur_(Co-op_Missions)
+    wikilink: https://starcraft.fandom.com/wiki/Abathur_\(Co-op_Missions\)
```

Comment:
I don't like these escapes being in the config file, I'll create a helper function for this instead. Thanks for pointing it out though.

Reply:
Agreed, it's better to do it in the code.